RhythmTests/EasingTests.swift

```diff
@@ -0,0 +1,157 @@
+//
+//  EasingTests.swift
+//  Rhythm
+//
+//  Created by Brian Heim on 5/31/17.
+//
+//
+
+import XCTest
+@testable import Rhythm
+
+class EasingTests: XCTestCase {
+
+    //****** General errors *******//
```

Comment:
Aight! I had seen 'mark' around but wasn't sure of its purpose, this is good to know.